### PR TITLE
A bunch of 22w13a stuff

### PR DESCRIPTION
**Pull request by NoahvdAa**
Maps a bunch of stuff from the 22w13a snapshot. I scrolled through the diff and mapped anything that had looked like it had a noticeable change (excluded most new classes and stuff that looked like it would conflict with other PRs)

### CODE REVIEW

File: mappings/net/minecraft/data/server/HusbandryTabAdvancementGenerator.mapping [hunk 1]
```diff
@@ -3,6 +3,7 @@ CLASS net/minecraft/unmapped/C_thbfaetr net/minecraft/data/server/HusbandryTabAd
 	FIELD f_kiylfurm FISH_BUCKET_ITEMS [Lnet/minecraft/unmapped/C_vorddnax;
 	FIELD f_kvdwtxnc AXE_ITEMS [Lnet/minecraft/unmapped/C_vorddnax;
 	FIELD f_seatqrby BREEDABLE_ANIMALS [Lnet/minecraft/unmapped/C_ogavsvbr;
+	FIELD f_yrauwtms INDIRECTLY_BEEDABLE_ANIMALS [Lnet/minecraft/unmapped/C_ogavsvbr;
```
Comment (triphora) [inline mappings/net/minecraft/data/server/HusbandryTabAdvancementGenerator.mapping]: What is a beedable

Did you mean: breedable?

```suggestion
	FIELD f_yrauwtms INDIRECTLY_BREEDABLE_ANIMALS [Lnet/minecraft/unmapped/C_ogavsvbr;
```

Comment (NoahvdAa) [reply mappings/net/minecraft/data/server/HusbandryTabAdvancementGenerator.mapping]: I did, thanks.

**Review by triphora — APPROVED** [review]

File: mappings/net/minecraft/world/event/EntityPositionSource.mapping [hunk 2]
```diff
@@ -1,2 +1,35 @@
 CLASS net/minecraft/unmapped/C_jwdpbxxx net/minecraft/world/event/EntityPositionSource
+	FIELD f_ghrcjuvk CODEC Lcom/mojang/serialization/Codec;
+	FIELD f_jepyhqzp yOffset F
+	FIELD f_uhlfletp eitherEntityOrUUIDOrEntityId Lcom/mojang/datafixers/util/Either;
```
Comment (IotaBread) [inline mappings/net/minecraft/world/event/EntityPositionSource.mapping]: I don't think being so specific here is useful, `entityId` would work imo

File: mappings/net/minecraft/world/gen/structure/TerrainAdjustment.mapping [hunk 3]
```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/unmapped/C_djexrdop net/minecraft/world/gen/structure/TerrainAdjustment
+	FIELD f_awcqkimx BEARD_BOX Lnet/minecraft/unmapped/C_djexrdop;
+	FIELD f_fykwreok BEARD_THIN Lnet/minecraft/unmapped/C_djexrdop;
```
Comment (IotaBread) [inline mappings/net/minecraft/world/gen/structure/TerrainAdjustment.mapping]: We use structure weight instead of beard iirc

**Review by IotaBread — COMMENTED** [review]
Looks good, aside from the comments

**Review by NoComment1105 — APPROVED** [review]
Looks good pending other comment resolution

**Review by Bubblie01 — APPROVED** [review]

File: mappings/net/minecraft/world/event/EntityPositionSource.mapping [hunk 4]
```diff
@@ -1,2 +1,35 @@
 CLASS net/minecraft/unmapped/C_jwdpbxxx net/minecraft/world/event/EntityPositionSource
+	FIELD f_ghrcjuvk CODEC Lcom/mojang/serialization/Codec;
+	FIELD f_jepyhqzp yOffset F
+	FIELD f_uhlfletp eitherEntityOrEntityId Lcom/mojang/datafixers/util/Either;
+	METHOD <init> (Lcom/mojang/datafixers/util/Either;F)V
+		ARG 1 eitherEntityOrUUIDOrEntityId
```
Comment (IotaBread) [inline mappings/net/minecraft/world/event/EntityPositionSource.mapping]:
```suggestion
	FIELD f_uhlfletp entityOrId Lcom/mojang/datafixers/util/Either;
	METHOD <init> (Lcom/mojang/datafixers/util/Either;F)V
		ARG 1 entityOrId
```

**Pull request merged**